packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -424,6 +446,82 @@ const TagSubforumPage2 = ({classes}: {
   ) : <></>;
   const rightSidebarComponents = [welcomeBoxComponent, <SidebarSubtagsBox tagId={tag._id} className={classes.sidebarBoxWrapper} key={`subtags_box`}/>];
 
+  const SubforumFeed = () => {
+    const refetchRef = useRef<null|(()=>void)>(null);
+    const refetch = useCallback(() => {
+      if (refetchRef.current)
+        refetchRef.current();
+    }, [refetchRef]);
+    const commentNodeProps = {
+      treeOptions: {
+        postPage: true,
+        refetch,
+        tag,
+      },
+      startThreadTruncated: true,
+      isChild: false,
+      enableGuidelines: false,
+      displayMode: "minimalist" as const,
+    };
+    const sortBy: SubforumSorting = isSubforumSorting(query.sortedBy) ? query.sortedBy : defaultSubforumSorting;
+    return <div className={classNames(classes.centralColumn, classes.feedWrapper)}>
+      <div className={classes.feedHeader}>
+        <div className={classes.feedHeaderButtons}>
+          <SectionButton>
+            <AddBoxIcon /> New Post
+          </SectionButton>
+          <SectionButton>
+            <AddBoxIcon /> New Discussion
+          </SectionButton>
+        </div>
+        <PostsListSortDropdown value={query.sortedBy} options={subforumSortings} />
+      </div>
+      <MixedTypeFeed
+        firstPageSize={10}
+        pageSize={20}
+        refetchRef={refetchRef}
+        resolverName={`Subforum${subforumSortingToResolverName(sortBy)}Feed`}
+        sortKeyType={subforumSortingTypes[sortBy]}
+        resolverArgs={{
+          tagId: 'String!',
+          af: 'Boolean',
+        }}
+        resolverArgsValues={{
+          tagId: tag._id,
+          af: false,
+        }}
+        fragmentArgs={{}}
+        fragmentArgsValues={{}}
+        renderers={{
+          tagSubforumPosts: {
+            fragmentName: "PostsList",
+            render: (post: PostsList) => (
+              <div className={classes.feedPostWrapper}>
+                <RecentDiscussionThread
```

Comment:
I just co-opted this component for now since it seems to do ~exactly what we want, but I think it's quite likely we'll eventually want something custom here.